addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java

```diff
@@ -247,7 +247,7 @@ public void scan() {
                 try {
                     hackedMessage.setCookieParams(originalMsg.getCookieParams());
                 } catch (Exception e) {
-                    log.warn("Could not set the cookies from the base request: {}", e);
+                    log.warn("Could not set the cookies from the base request: ", e);
```

Comment:
Was this intended? The first one adds the exception's message to the log message, whereas the second one includes the stack trace of the exception.

(Same goes for the other changes like this one in this PR).

Reply:
Hmmm Sonar doesn't seem to recognize the difference, it says this is insufficient arguments in the original form. I'll tweak them to literally `e.getMessage()` I guess.

Reply:
I dont think that will do what we want, ie show the stack trace.
How about `log.warn("Could not set the cookies from the base request: {}", e.getMessage(), e);` ?